FlightReservation/screens/registerFlightScreens/originScreen.js

```diff
@@ -0,0 +1,8 @@
+import React, {useState} from 'react';
+
+const OriginScreen = () => {
+
+}
+
+
+export default OriginScreen;
```

Comment:
What is the point of this file?

Reply:
it was to set to host a new screen but, we will delete it

Reply:
Done!